prow/plugins/config.go

```diff
@@ -1300,8 +1292,8 @@ func (c *Configuration) Validate() error {
 		logrus.Warn("no plugins specified-- check syntax?")
 	}
 
-	if c.Owners.LabelsBlackList != nil && c.Owners.LabelsDenyList != nil {
-		return errors.New("labels_blacklist and labels_denylist cannot be both supplied")
+	if c.Owners.LabelsDenyList != nil {
+		return errors.New("labels_denylist cannot be supplied")
```

Comment:
this might be incorrect. `labels_denylist` could be supplied, am I missing something?

Reply:
Thank you for bringing this up. I think `labels_denylist` could be supplied. So I will make the required changes.